Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_armblade.yml

```
@@ -0,0 +1,50 @@
+  - type: Item
+    size: Ginormous
+    sprite: _Goobstation/Changeling/arm_blade.rsi
+  - type: Prying
+    speedModifier: 2
+    pryPowered: true
+  - type: UseDelay # For insta prying
+    delay: 1
+  - type: Unremoveable
+  - type: Tool
+    qualities:
+    - Slicing
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)